### PR TITLE
Transfer Library Bug

### PR DESCRIPTION
**Pull request by GeorgWa**
- handle cases where search output doesn't have calibrated values (`rt_calibrated`...) gracefully
- some slight improvements to fix warnigns along the way

### CODE REVIEW

File: alphadia/outputaccumulator.py [hunk 1]
```diff
@@ -99,38 +100,47 @@ def parse_output_folder(
...
+        if optional_precursor_columns is None:
+            optional_precursor_columns = [
+                "rt_calibrated",
+                "mz_calibrated",
+            ]
+
         psm_df = pd.read_parquet(os.path.join(folder, "psm.parquet"))
         frag_df = pd.read_parquet(os.path.join(folder, "frag.parquet"))
 
         assert set(
```
Comment (mschwoer) [inline alphadia/outputaccumulator.py:133]: https://google.github.io/styleguide/pyguide.html#244-decision

File: alphadia/outputaccumulator.py [hunk 2]
```diff
@@ -99,38 +100,47 @@ def parse_output_folder(
...
-            selected_precursor_columns
+            mandatory_precursor_columns
         ).issubset(
             psm_df.columns
-        ), f"selected_precursor_columns must be a subset of psm_df.columns didnt find {set(selected_precursor_columns) - set(psm_df.columns)}"
-        psm_df = psm_df[selected_precursor_columns]
-        # validate.precursors_flat_from_output(psm_df)
+        ), f"mandatory_precursor_columns must be a subset of psm_df.columns didnt find {set(mandatory_precursor_columns) - set(psm_df.columns)}"
+
+        available_columns = list(
```
Comment (mschwoer) [inline alphadia/outputaccumulator.py:139]: `sorted()`? :-)

File: alphadia/workflow/peptidecentric.py [hunk 3]
```diff
@@ -260,6 +260,30 @@ def norm_to_rt(
         else:
             raise ValueError(f"Unknown norm_rt_mode {mode}")
 
+    @property
```
Comment (mschwoer) [inline alphadia/workflow/peptidecentric.py:263]: are these properties also to be accessed from outside the class?

Comment (GeorgWa) [reply alphadia/workflow/peptidecentric.py:263]: No, most likely not.

Comment (mschwoer) [reply alphadia/workflow/peptidecentric.py:263]: then I would suggest settings these values as instance variables or simple methods

File: alphadia/workflow/peptidecentric.py [hunk 4]
```diff
@@ -260,24 +260,30 @@ def norm_to_rt(
         else:
             raise ValueError(f"Unknown norm_rt_mode {mode}")
 
-    @property
-    def precursor_mz_column(self):
+    def get_precursor_mz_column(self):
```
Comment (mschwoer) [inline alphadia/workflow/peptidecentric.py:263]: could all these be private?

**Review by mschwoer — APPROVED** [review]

**Pull request merged**